unblob/models.py

```diff
@@ -88,6 +100,7 @@ class ValidChunk(Chunk):
 
     handler: "Handler" = attr.ib(init=False, eq=False)
     is_encrypted: bool = attr.ib(default=False)
```

Comment:
just wondering that now we have metadata, is_encrypted is metadata or still a member of valid chunk?

Reply:
I would move it to metadata and adapt handlers accordingly.

Reply:
After some code writing I think `is_encrypted` is better there as an attribute than within `metadata`.